cookbook/docs/getting_started/package_register.md

```diff
@@ -0,0 +1,326 @@
+you can invoke it like so:
+
+```{prompt} bash $
+pyflyte run --remote workflows/example.py wf --name "Gilgamesh"
+```
+
+````{div} shadow p-3 mb-8 rounded
+
+**Expected Output:** A URL to the workflow execution on your demo Flyte cluster:
+
+```{code-block}
+Go to http://localhost:30080/console/projects/flytesnacks/domains/development/executions/<execution_name> to see execution in the console.
+```
+
+Where ``<execution_name>`` is a unique identifier for the workflow execution.
+
+````
+
+`pyflyte run` will not only register the specified workflow `wf`, it will also
+run it with the supplied arguments. As you can see from the expected output, you
+can visit the link to the Flyte console to see the progress of your running
+execution.
+
+```{important}
+Currently, `pyflyte run` only supports Flyte workflows that are in self-contained
+scripts, meaning that it shouldn't import any other user-defined modules that
+contain other tasks or workflows.
+```
+
+## Iterating on a Flyte Project
+
+One of Flyte's benefits is its functional design, which means that you can
+import and reuse tasks and workflows like you would Python functions when you
+organize your code into meaningful sets of modules and subpackages.
+
+When you move past a single script that contains your workflows, use the
+`pyflyte register` command to register all the tasks and workflows contained
+in the specified directory or file:
+
+```{prompt} bash $
+pyflyte register workflows
+```
+
+````{div} shadow p-3 mb-8 rounded
+
+**Expected Output:**
+
+```{code-block} bash
+Successfully serialized 4 flyte objects
+Found and serialized 4 entities
+  Registering workflows.example.say_hello....done, TASK with version sdYMF0jAkhDh_KA1IMAFYA==.
+  Registering workflows.example.greeting_length....done, TASK with version sdYMF0jAkhDh_KA1IMAFYA==.
+  Registering workflows.example.wf....done, WORKFLOW with version sdYMF0jAkhDh_KA1IMAFYA==.
+  Registering workflows.example.wf....done, LAUNCH_PLAN with version sdYMF0jAkhDh_KA1IMAFYA==.
+Successfully registered 4 entities
+```
+
+````
+
+By default, `pyflyte register` uses a [default Docker image](https://ghcr.io/flyteorg/flytekit)
+that's maintained by the Flyte team, but you can use your own Docker image by
+passing in the `--image` flag.
+
+For example, assuming that you want to use the latest Python 3.9 flytekit image,
+the explicit equivalent to the default image value would be something like:
+
+```{prompt} bash $
+pyflyte register workflows --image ghcr.io/flyteorg/flytekit:py3.9-latest
+```
+
+`pyflyte register` packages up your code through a mechanism called
+**fast registration**. At a high level, this will:
+
+1. Package and zip up the directory/file that you specify as the argument to
+   `pyflyte register`, which is the `workflows` directory in the example above.
+2. Register the Flyte package to the specified Flyte cluster, which is the
+   local Flyte demo cluster by default.
+
```

Comment:
Can we add a link to the https://docs.flyte.org/projects/flytekit/en/latest/design/clis.html page in case a user wants to drill down through the differences?